prql-java/java/pom.xml

```diff
@@ -4,10 +4,43 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.prql</groupId>
+    <groupId>org.prqllang</groupId>
     <artifactId>prql-java</artifactId>
     <version>0.2.1</version>
 
+    <name>${project.groupId}:${project.artifactId}</name>
+    <description>prql-compiler api for java</description>
+    <url>https://github.com/prql/prql</url>
+
+    <licenses>
+        <license>
+            <name>The Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Aljaž Mur Eržen</name>
+            <email>TODO</email>
```

Comment:
@aljazerzen @max-sixty @charlie-sanders Need your public information.

Reply:
Let's just have "PRQL Developers" "prql@prql-lang.org" — it's the work of many more than just us